foods.html

```diff
@@ -10,6 +10,14 @@
 <body>
   <h1>Manage Foods</h1>
 
```

Comment:
I would wrap this in a Div so we can style this

Reply:
Should be able to style your `h1` tags without wrapping them in a `div`. Both are block elements (stretch the whole width of the page), so not positive how much advantage you get from a div.

If you wanted to put other elements on the same line (e.g. wanted to have some navigation buttons to the right of the `h1`, then I could see wrapping *all of that* in a `div`, but just the `h1` can probably stand on its own.

Unless you were talking about the `form` below? But similar comment there. Believe you should be able to style the `form` element directly.

I suppose you could wrap all of the content of the page in a `div` with a `main` class or something of that sort so that you could center content on the page, etc.